### ISSUE TITLE
Stream with empty url

### ISSUE THREAD

**Issue by anime-src** [Stremio/stremio-addon-sdk#230]
Hi, not sure if ticket is on sdk or stremio package for android tv, 
but in builder.defineStreamHandler express app, if you define multiple streams like

```
 streams: [
    {
      name: 'Name1',
      title: 'My Title 2',
      url: '',
    },
    {
       name: 'Name2',
      title: 'My Title 2',
      url: 'https://mylink2.m3u8',
    },
    {
         name: 'Name3',
      title: 'My Title 3',
      url: 'https://mylink3.m3u8',
    }
```
and one empty url,  stremio on android tv (google app store version) will return no streams found
that behavior was not present on windows, web, or android tv package downloaded from stremio site

**Comment by jaruba** [comment]
Are you sure it works in Stremio Web? from this link: https://web.stremio.com/

I'm asking because both the android tv app and the web app use the new core, while the mobile and desktop apps use the old core.

The new core is built with Rust and is type safe (see the `stremio-core repository) which can cause deserialization errors if addons don't provide data correctly.

**Comment by anime-src** [comment]
Yes tested with web version and i did get streams, now i think web version only showed streams that have url defined, need to confirm this, but the one from google app store showed no streams found, stream with empty url was always first one in streams list, had addon running localy so i can see json returning from addon when android tv was requesting, quick fix was to remove streams where url is empty
